.github/workflows/python-client-release-to-pypi.yml

```
@@ -0,0 +1,26 @@
+name: Release Python Client to pypi
```

```suggestion
name: Release Python Client to PYPI
```

nitpick.  Just some consistent formatting for the workflow names.